Bazooka.lua

```diff
@@ -56,7 +56,7 @@ local IsAltKeyDown = _G.IsAltKeyDown
 local IsShiftKeyDown = _G.IsShiftKeyDown
 local IsModifierKeyDown = _G.IsModifierKeyDown
 local GetCursorPosition = _G.GetCursorPosition
-local GetAddOnInfo = _G.GetAddOnInfo
+local GetAddOnInfo = C_AddOns.GetAddOnInfo
```

Comment:
While the addon functions without this specific change, I concur: this is likely best to update now. Note that 2675 is needed.

Reply:
Well, it threw an error for me. Didn't check if the addon fully worked nonetheless ;)